### PR TITLE
WIP: Remove openstack bits

### PR DESCRIPTION
**Pull request by sshnaidm**
remove all openstack files after it's moved to collection openstack.cloud

hold-on

### CODE REVIEW

File: docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst [hunk 1]
```diff
@@ -1,60 +0,0 @@
-.. _OpenStack_module_development:
```
Comment (gundalow) [inline docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst:1]: @acozine @abadger Should dev_guide documentation guides stay in ansible/ansble?
@sshnaidm Does this documentation exist elsewhere?

**Review by gundalow — CHANGES_REQUESTED** [review]
Marking as requires changes as we can't merge this till Collection docs build work is done.

File: docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst (hunk 1 above)
Comment (sshnaidm) [reply docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst:1]: @gundalow not in openstack repo, but as I see it's mostly incorrect atm. So anyways we'll need to write a new one. I'll submit a patch with this file and we'll start to review it. In the current state it's totally outdated.

Comment (sshnaidm) [reply docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst:1]: @gundalow submitted in https://review.opendev.org/#/c/704558/

Comment (gundalow) [reply docs/docsite/rst/dev_guide/platforms/openstack_guidelines.rst:1]: Perfect, thank you.

**Comment by gundalow** [comment]
So the world has moved on since we originally talked about this.

This will happen automatically when the migrate.py script gets run on Monday

**Pull request closed without merging**